types/jwplayer/jwplayer-tests.ts

```diff
@@ -1,2 +1,46 @@
-jwplayer('id').resize(40, 40);
```

Comment:
Why are the resize test cases deleted?

Reply:
Not deleted. Just moved to other functions that return JWPlayer

Reply:
@martin-badin my bad. I totally missed that 😁